packages/client/src/types/unified.ts

```diff
@@ -550,6 +550,8 @@ export type MsgsResponse = {
 
 export type BridgeType = 'IBC' | 'AXELAR' | 'CCTP' | 'HYPERLANE' | 'OPINIT';
 
+export type ChainType = 'cosmos' | 'evm' | 'svm' | 'sei';
```

Comment:
sei is not a chain type, please remove sei

Reply:
we treat it like a chainType here. should this be refactored? 

https://github.com/skip-mev/skip-go/blob/4629bcaeadb3430d3e722f393638b254c74dcbc8/packages/widget-v2/src/hooks/useWalletList.tsx#L25

Reply:
ok resolved this here: https://github.com/skip-mev/skip-go/pull/334/commits/3bd27716e42c64c5fc6f3c6e7e3402ab4144f40e